src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -76,7 +76,10 @@
 import com.puppycrawl.tools.checkstyle.filters.SuppressionFilter;
 import com.puppycrawl.tools.checkstyle.utils.CommonUtils;
 
-public class CheckerTest extends BaseCheckTestSupport {
+/**
+ * @noinspection ClassWithTooManyDependencies
+ */
+public class CheckerTest extends AbstractModuleTestSupport {
```

Comment:
yes, looks like bug, I failed to find a solution for this.
@subkrish , please update limit to be `34`.